os/drivers/lcd/st7785.c

```diff
@@ -0,0 +1,50 @@
+/****************************************************************************
+ *
+ * Copyright 2025 Samsung Electronics All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ****************************************************************************/
+
+#include <tinyara/lcd/st7785.h>
+#include <debug.h>
+
+extern int send_init_cmd(struct mipi_lcd_dev_s *priv, lcm_setting_table_t *table);
+extern int set_return_packet_len(struct mipi_lcd_dev_s *priv, u8 rx_len);
+extern int read_response(struct mipi_lcd_dev_s *priv, lcm_setting_table_t command, u8 *rxbuf, u8 rx_len);
+
+int check_lcd_vendor_send_init_cmd(struct mipi_lcd_dev_s *priv)
+{
+	/* Check Vendor */
+	lcm_setting_table_t read_display_cmd = {0x04, 0, {0x00}};
+	uint8_t rxbuf[3] = {0xFF,0xFF,0xFF};

```

Comment:
1. space after `,`
2. Is there any chance to be changed the number of index later? Even if not, `sizeof(rxbuf) / sizeof(rxbuf[0]` is used 3 times. How about calculating it once and use it all?